### PR TITLE
feat(editor): Workflow history [WIP] - Add cloned workflow link to success toast message (no-changelog)

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/composables/useToast.ts [hunk 1]
```diff
@@ -19,17 +19,14 @@ export function useToast() {
...
-	) {
+	function showMessage(messageData: NotificationOptions, track = true) {
 		messageData = { ...messageDefaults, ...messageData };
-		messageData.message = messageData.message
-			? sanitizeHtml(messageData.message)
-			: messageData.message;
+		messageData.message =
+			typeof messageData.message === 'string'
+				? sanitizeHtml(messageData.message)
+				: messageData.message;
```
Comment (ivov) [inline packages/editor-ui/src/composables/useToast.ts:27]: Just curious, what is `message` when not a string?

**Review by ivov — COMMENTED** [review]
Code looks great! Findings:

1. Cloned workflow is missing pinned data and tags. Not sure if intended.
2. Cloned workflow should have same version history as original workflow up to this point? It's rather confusing to see no versions when looking at the resulting workflow.
3. Cloning a workflow twice (in a row, or after a while) triggers [this error](https://share.cleanshot.com/ZkqPw1hx). I'd expect us to handle it like workflow duplication.
4. Cloning a cloned workflow results in [weird naming](https://share.cleanshot.com/rJW1SxY8).
5. Cloning an active workflow deactivates the copy. Maybe we should clarify.
6. Toast for `Successfully cloned workflow version` does not clear when navigating away. Not sure if intended.

File: packages/editor-ui/src/composables/useToast.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/editor-ui/src/composables/useToast.ts:27]: The problem was here we could only pass string and I needed a link (`<a href="">`) and sometimes you need more HTML and the original ElementUI component can also accept `VNODE` or a function returning a `VNODE` so I wanted to make it fully compatible

See `message` in the [docs](https://element-plus.org/en-US/component/message.html#options)

**Comment by cstuncsik** [comment]
Fixed number 3. the rest are more like design question (either graphic or functional)

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.14.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.14.0)

**Pull request merged**